### PR TITLE
Remove apt and yum dependencies

### PR DESCRIPTION
**Pull request by jon918**
Remove callouts to specific apt and yum cookbook versions to simplify dependency resolution.

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -1,11 +1,14 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-source "https://supermarket.chef.io"
+source "https://supermarket.chef.io"
```
Comment (gamache) [inline Berksfile]: The road to hell is paved with trailing whitespace

Comment (jon918) [inline Berksfile]: Hi pete! thanks. 👍

Comment (jr314159) [inline Berksfile]: At the mbta we have standardized on two trailing spaces per line

Comment (jon918) [inline Berksfile]: Crushing those insights guys

**Comment by mrdziuban** [comment]
:shipit:

**Pull request merged**